docs/course_materials/exercises/04_Compound_data_types/homework_dictionary.md

```diff
@@ -7,7 +7,7 @@
 
 Create a file *countries.py*
 
-- Create a new dictionary with the Udi number of flights to each country:  “Italy”:2 “Spain”:3  “Israel”:1
+- Create a new dictionary with Udi's flights number to each country:  “Italy”:2 “Spain”:3  “Israel”:1
```

Comment:
split into 2 lines:
Create a new dictionary with Udi's flights' number to each country: <br>
“Italy”:2 “Spain”:3  “Israel”:1